x-pack/solutions/security/packages/navigation/src/constants.ts

```diff
@@ -94,6 +94,7 @@ export enum ExternalPageName {
   managementFiles = 'management:filesManagement',
   managementSpaces = 'management:spaces',
   managementSettings = 'management:settings',
+  managementMonitoring = 'management:stack_monitoring',
```

Comment:
This value is not arbitrary, it's explained in the comment above.
This should be:

```suggestion
  managementMonitoring = 'monitoring:',
```

Reply:
Thanks, I actually tried `monitoring:`, `management:stack_monitoring` and `management:monitoring` and none worked because of the missing step of adding it to https://github.com/elastic/kibana/blob/9a06509972a976b471cef5af496690ff6200223e/x-pack/solutions/security/plugins/security_solution/public/app/solution_navigation/categories.ts#L67-L74